### PR TITLE
CPM-152: update job consuming

### PR DESCRIPTION
**Pull request by nmarniesse**
**Description**

In the nest release the way to run job daemons will change

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by BitOne — APPROVED** [review]

**Review by AymericPlanche — APPROVED** [review]

File: install_pim/docker/installation_docker.rst [hunk 1]
```diff
@@ -135,27 +135,28 @@ Running jobs
 
 Akeneo PIM implements a queue for the `jobs <../../import_and_export_data/index.html>`_, as a PHP daemon. This daemon is a Symfony command, that can only execute one job at a time. It does not consume any other job until the job is finished.
 
-You can launch several daemons to allow the execution of several jobs in parallel. A daemon checks every 5 seconds the queue and executes the next job in the queue.
+You can launch several daemons to allow the execution of several jobs in parallel. A daemon checks every second the queue and executes the next job in the queue.
```
Comment (mmetayer) [inline install_pim/docker/installation_docker.rst:138]:
```suggestion
You can launch several daemons to allow the execution of several jobs in parallel. A daemon checks the queue every second and executes the next job in the queue.
```

File: install_pim/docker/installation_docker.rst [hunk 2]
```diff
@@ -135,27 +135,28 @@ Running jobs
...
 
 .. code-block:: bash
 
-   APP_ENV=dev docker-compose run --rm php php bin/console akeneo:batch:job-queue-consumer-daemon --run-once
+   APP_ENV=dev make start-job-worker O="--limit=1"
 
 .. warning::
 
    Before stopping or destroying your containers, remember to first stop this daemon if you launched it in the background, or you'll end up with a stuck FPM container, and will need to completely restart Docker.
+   Please note this command sends a stop signal to all the consumers. If a job is running while the stop signal is sent, the consumer will wait for the end of the running job before ending.
```
Comment (mmetayer) [inline install_pim/docker/installation_docker.rst:155]:
```suggestion
   Please note that this command sends a stop signal to all the consumers. If a job is running while the stop signal is sent, the consumer will wait for the end of the running job before ending.
```

File: install_pim/manual/daemon_queue.rst [hunk 3]
```diff
@@ -13,51 +13,58 @@ You can launch several daemon processes to execute multiple jobs in parallel.
 
 Also, the daemon processes could be run on several instance of the PIM, using the same MySQL database.
 
-This queue allows `horizontal scalability <https://en.wikipedia.org/wiki/Scalability#Horizontal_and_vertical_scaling>`_ of the PIM.
+These queues allows `horizontal scalability <https://en.wikipedia.org/wiki/Scalability#Horizontal_and_vertical_scaling>`_ of the PIM.
```
Comment (mmetayer) [inline install_pim/manual/daemon_queue.rst:16]:
```suggestion
These queues allow `horizontal scalability <https://en.wikipedia.org/wiki/Scalability#Horizontal_and_vertical_scaling>`_ of the PIM.
```

**Review by mmetayer — APPROVED** [review]

**Comment by nmarniesse** [comment]
TODO: add SF coimmand to migrate jobs

**Pull request merged**